www/src/components/feedback-widget/feedback-form.js

```diff
@@ -71,6 +71,8 @@ const textareaStyles = {
   height: `5.5rem`,
   mt: 1,
   mb: 4,
+  lh: `1.25rem`,
+  padding: `0.5rem 0.75rem`,
```

Comment:
Similar to above, we should be able to do

```
px: 3,
py: 2,
```

here to use theme's `space` tokens. They are not properly documented yet for the use with `theme-ui` and the `sx` prop, but I think you'll work through that: https://www.gatsbyjs.org/guidelines/design-tokens/